src/redux/wallet/selectors/models.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import { createSelector } from 'reselect'
+import { DUCK_MAIN_WALLET } from 'redux/mainWallet/actions'
+import { DUCK_MULTISIG_WALLET } from 'redux/multisigWallet/actions'
+import { DUCK_MARKET } from 'redux/market/action'
+import { DUCK_TOKENS } from 'redux/tokens/actions'
+import { getCurrentWallet } from 'redux/wallet/actions'
+
+/**
+ * SIMPLE SELECTORS
+ * ==============================================================================
+ */
+
+export const getMainWallet = (state) => {
```

Comment:
I "suspect" that getMainWallet and getMultisigWallets are implemented multiple times in different places.

Reply:
No, only one.